### ISSUE TITLE
rdf-writer (LdioHttpOut) crashes when JSON-LD context is 1.1

### ISSUE THREAD

**Issue by kkostov** [Informatievlaanderen/VSDS-Linked-Data-Interactions#473]
**Describe the bug**
It has been reported that the rdf writer does not support JSON-LD using context `https://geojson.org/geojson-ld/geojson-context.jsonld`

**To Reproduce**

Version: `ldes/ldi-orchestrator:1.13.0-SNAPSHOT`

Example configuration:

```yaml
outputs:
  - name: be.vlaanderen.informatievlaanderen.ldes.ldio.LdioHttpOut
    config:
      rdf-writer:
        content-type: application/ld+json
        frame: |
          { "@context": "https://geojson.org/geojson-ld/geojson-context.jsonld", 
            "@type": "https://vocabulary.uncefact.org/LogisticsLocation" }
      endpoint: http://host.docker.internal:3000/api/entity/
```

Stacktrace:

```
2024-02-01 10:24:26 2024-02-01T09:24:26.969Z  INFO 1 --- [pool-6-thread-1] l.c.s.StartingTreeNodeFinder             : Parsing response for:
http://participant1-connector:9291/public
2024-02-01 10:24:27 2024-02-01T09:24:27.867Z ERROR 1 --- [pool-5-thread-1] b.v.i.ldes.ldio.LdioHttpOut              : ERROR - when='output', problem='com.github.jsonldjava.core.JsonLdError: invalid term definition: 1.1'
2024-02-01 10:24:27 2024-02-01T09:24:27.869Z ERROR 1 --- [pool-5-thread-1] b.v.i.ldes.ldio.LdioLdesClient           : ERROR - when='client-pipeline:processModel', problem='com.github.jsonldjava.core.JsonLdError: invalid term definition: 1.1'
2024-02-01 10:24:27 2024-02-01T09:24:27.870Z ERROR 1 --- [pool-5-thread-1] b.v.i.ldes.ldio.LdioLdesClient           : LdesClientRunner FAILURE
2024-02-01 10:24:27 
2024-02-01 10:24:27 org.apache.jena.riot.RiotException: com.github.jsonldjava.core.JsonLdError: invalid term definition: 1.1
2024-02-01 10:24:27     at org.apache.jena.riot.writer.JsonLD10Writer.serialize(JsonLD10Writer.java:186) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at org.apache.jena.riot.writer.JsonLD10Writer.write(JsonLD10Writer.java:138) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at org.apache.jena.riot.writer.JsonLD10Writer.write(JsonLD10Writer.java:144) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at org.apache.jena.riot.system.RiotLib$WriterAdapter.write(RiotLib.java:432) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at org.apache.jena.riot.RDFWriter.write$(RDFWriter.java:296) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at org.apache.jena.riot.RDFWriter.output(RDFWriter.java:255) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at org.apache.jena.riot.RDFWriter.output(RDFWriter.java:198) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at org.apache.jena.riot.RDFWriterBuilder.output(RDFWriterBuilder.java:207) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldi.rdf.formatter.JsonLdFrameWriter.writeToOutputStream(JsonLdFrameWriter.java:34) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.LdioHttpOut.accept(LdioHttpOut.java:38) ~[ldio-http-out-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.LdioHttpOut.accept(LdioHttpOut.java:19) ~[ldio-http-out-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.config.LdiOutputLogger.lambda$accept$0(LdiOutputLogger.java:27) ~[!/:na]
2024-02-01 10:24:27     at io.micrometer.observation.Observation.lambda$observe$0(Observation.java:493) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-02-01 10:24:27     at io.micrometer.observation.Observation.observeWithContext(Observation.java:603) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-02-01 10:24:27     at io.micrometer.observation.Observation.observe(Observation.java:492) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.config.LdiOutputLogger.accept(LdiOutputLogger.java:25) ~[!/:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.config.LdiOutputLogger.accept(LdiOutputLogger.java:10) ~[!/:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.services.LdioSender.lambda$apply$1(LdioSender.java:57) ~[!/:na]
2024-02-01 10:24:27     at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:290) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:754) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:686) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:159) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:173) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:765) ~[na:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.services.LdioSender.apply(LdioSender.java:57) ~[!/:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.services.ComponentExecutorImpl.transformLinkedData(ComponentExecutorImpl.java:17) ~[!/:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.types.LdioInput.lambda$processModel$1(LdioInput.java:73) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at io.micrometer.observation.Observation.lambda$observe$0(Observation.java:493) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-02-01 10:24:27     at io.micrometer.observation.Observation.observeWithContext(Observation.java:603) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-02-01 10:24:27     at io.micrometer.observation.Observation.observe(Observation.java:492) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.types.LdioInput.processModel(LdioInput.java:71) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at be.vlaanderen.informatievlaanderen.ldes.ldio.LdioLdesClient.run(LdioLdesClient.java:55) ~[ldio-ldes-client-connector-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[na:na]
2024-02-01 10:24:27     at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[na:na]
2024-02-01 10:24:27     at java.base/java.lang.Thread.run(Thread.java:840) ~[na:na]
2024-02-01 10:24:27 Caused by: com.github.jsonldjava.core.JsonLdError: invalid term definition: 1.1
2024-02-01 10:24:27     at com.github.jsonldjava.core.Context.createTermDefinition(Context.java:349) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at com.github.jsonldjava.core.Context.parse(Context.java:289) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at com.github.jsonldjava.core.Context.parse(Context.java:217) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at com.github.jsonldjava.core.Context.parse(Context.java:150) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at com.github.jsonldjava.core.Context.parse(Context.java:305) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at com.github.jsonldjava.core.JsonLdApi.expand(JsonLdApi.java:550) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at com.github.jsonldjava.core.JsonLdApi.expand(JsonLdApi.java:1027) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at com.github.jsonldjava.core.JsonLdProcessor.expand(JsonLdProcessor.java:139) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at com.github.jsonldjava.core.JsonLdProcessor.frame(JsonLdProcessor.java:312) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-02-01 10:24:27     at org.apache.jena.riot.writer.JsonLD10Writer.toJsonLDJavaAPI(JsonLD10Writer.java:221) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     at org.apache.jena.riot.writer.JsonLD10Writer.serialize(JsonLD10Writer.java:177) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-02-01 10:24:27     ... 42 common frames omitted
2024-02-01 10:24:27 
```

**Expected behavior**
It should be possible to use the current version of JSON-LD.

**Comment by alperbasak** [comment]
Environment:
- ldes/ldes-server:2.8.0-SNAPSHOT
- ldes/ldi-orchestrator:1.13.0

ldio-config: (using connector config, but see the frame)
```
    - name: nodes-pipeline
      description: "nodes LdioLdesClientConnector in, http out pipeline."
      input:
        name: be.vlaanderen.informatievlaanderen.ldes.ldio.LdioLdesClientConnector
        config:
          url: http://participant1-connector:9291/public/nodes
          connector-transfer-url: http://participant1-connector:9193/management/v2/transferprocesses
          proxy-url-to-replace: http://localhost:8081/nodes
          proxy-url-replacement: http://participant1-connector:9291/public/nodes
          source-format: application/ld+json
      outputs:
        - name: be.vlaanderen.informatievlaanderen.ldes.ldio.LdioConsoleOut
          config:
            rdf-writer:
              content-type: application/ld+json
              frame: |
                { "@context": "https://geojson.org/geojson-ld/geojson-context.jsonld",
                  "@type": "https://vocabulary.uncefact.org/LogisticsLocation" }
            endpoint: https://webhook.site/00d3288d-4866-4155-a18f-a017e3e1f47a
```


Trying to 
`curl -X POST -H "Content-Type: application/ld+json" -d @data/node.template.json http://ldes-server:8081/nodes`

where the `node.template.json`:

```
{
  "@context": [
    "https://vocabulary.uncefact.org/unece-context.jsonld",
    "https://geojson.org/geojson-ld/geojson-context.jsonld",
    {
      "isVersionOf": "http://purl.org/dc/terms/isVersionOf",
      "generatedAtTime": {
        "@id": "http://www.w3.org/ns/prov#generatedAtTime",
        "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
      }
    }
  ],
  "type": "LogisticsLocation",
  "isVersionOf": "unece:identifier/node/1",
  "generatedAtTime": "2020-05-20T10:00:00Z",
  "id": "unece:identifier/node/1/1",
  "name": "k1700",
  "statusCode": "0",
  "geometry": {
    "type": "Point",
    "coordinates": [
      4.9041,
      52.3676
    ]
  },
  "servicingSpecifiedParty": {
    "type": "LocationParty",
    "id": "unece:identifier/servicingSpecifiedParty/1",
    "name": "DP-World"
  }
}

```

results in:

```
2024-03-13 11:08:13 2024-03-13T10:08:13.950Z  INFO 1 --- [pool-6-thread-1] l.c.s.StartingTreeNodeFinder             : Parsing response for: http://participant1-connector:9291/public/nodes
2024-03-13 11:08:14 2024-03-13T10:08:14.210Z ERROR 1 --- [pool-5-thread-1] b.v.i.ldes.ldio.LdiConsoleOut            : ERROR - when='output', problem='com.github.jsonldjava.core.JsonLdError: invalid term definition: 1.1'
2024-03-13 11:08:14 2024-03-13T10:08:14.212Z ERROR 1 --- [pool-5-thread-1] b.v.i.ldes.ldio.LdioLdesClient           : ERROR - when='nodes-pipeline:processModel', problem='com.github.jsonldjava.core.JsonLdError: invalid term definition: 1.1'
2024-03-13 11:08:14 2024-03-13T10:08:14.212Z ERROR 1 --- [pool-5-thread-1] b.v.i.ldes.ldio.LdioLdesClient           : LdesClientRunner FAILURE
2024-03-13 11:08:14 
2024-03-13 11:08:14 org.apache.jena.riot.RiotException: com.github.jsonldjava.core.JsonLdError: invalid term definition: 1.1
2024-03-13 11:08:14 at org.apache.jena.riot.writer.JsonLD10Writer.serialize(JsonLD10Writer.java:186) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 at org.apache.jena.riot.writer.JsonLD10Writer.write(JsonLD10Writer.java:138) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 at org.apache.jena.riot.system.RiotLib$WriterAdapter.write(RiotLib.java:437) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 at org.apache.jena.riot.RDFWriter.write$(RDFWriter.java:306) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 at org.apache.jena.riot.RDFWriter.output(RDFWriter.java:269) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 at org.apache.jena.riot.RDFWriter.asString(RDFWriter.java:176) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 at org.apache.jena.riot.RDFWriterBuilder.asString(RDFWriterBuilder.java:225) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldi.rdf.formatter.JsonLdFrameWriter.write(JsonLdFrameWriter.java:29) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.LdiConsoleOut.accept(LdiConsoleOut.java:23) ~[ldio-console-out-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.LdiConsoleOut.accept(LdiConsoleOut.java:11) ~[ldio-console-out-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.config.LdiOutputLogger.lambda$accept$0(LdiOutputLogger.java:27) ~[!/:na]
2024-03-13 11:08:14 at io.micrometer.observation.Observation.lambda$observe$0(Observation.java:493) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-03-13 11:08:14 at io.micrometer.observation.Observation.observeWithContext(Observation.java:603) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-03-13 11:08:14 at io.micrometer.observation.Observation.observe(Observation.java:492) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.config.LdiOutputLogger.accept(LdiOutputLogger.java:25) ~[!/:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.config.LdiOutputLogger.accept(LdiOutputLogger.java:10) ~[!/:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.services.LdioSender.lambda$apply$1(LdioSender.java:57) ~[!/:na]
2024-03-13 11:08:14 at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:290) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:754) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:686) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:159) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:173) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:765) ~[na:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.services.LdioSender.apply(LdioSender.java:57) ~[!/:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.services.ComponentExecutorImpl.transformLinkedData(ComponentExecutorImpl.java:17) ~[!/:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.types.LdioInput.lambda$processModel$1(LdioInput.java:73) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at io.micrometer.observation.Observation.lambda$observe$0(Observation.java:493) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-03-13 11:08:14 at io.micrometer.observation.Observation.observeWithContext(Observation.java:603) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-03-13 11:08:14 at io.micrometer.observation.Observation.observe(Observation.java:492) ~[ldio-archive-file-in-jar-with-dependencies.jar:1.11.3]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.types.LdioInput.processModel(LdioInput.java:71) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at be.vlaanderen.informatievlaanderen.ldes.ldio.LdioLdesClient.run(LdioLdesClient.java:55) ~[ldio-ldes-client-connector-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[na:na]
2024-03-13 11:08:14 at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[na:na]
2024-03-13 11:08:14 at java.base/java.lang.Thread.run(Thread.java:840) ~[na:na]
2024-03-13 11:08:14 Caused by: com.github.jsonldjava.core.JsonLdError: invalid term definition: 1.1
2024-03-13 11:08:14 at com.github.jsonldjava.core.Context.createTermDefinition(Context.java:349) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at com.github.jsonldjava.core.Context.parse(Context.java:289) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at com.github.jsonldjava.core.Context.parse(Context.java:217) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at com.github.jsonldjava.core.Context.parse(Context.java:150) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at com.github.jsonldjava.core.Context.parse(Context.java:305) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at com.github.jsonldjava.core.JsonLdApi.expand(JsonLdApi.java:550) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at com.github.jsonldjava.core.JsonLdApi.expand(JsonLdApi.java:1027) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at com.github.jsonldjava.core.JsonLdProcessor.expand(JsonLdProcessor.java:139) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at com.github.jsonldjava.core.JsonLdProcessor.frame(JsonLdProcessor.java:312) ~[ldio-archive-file-in-jar-with-dependencies.jar:na]
2024-03-13 11:08:14 at org.apache.jena.riot.writer.JsonLD10Writer.toJsonLDJavaAPI(JsonLD10Writer.java:221) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 at org.apache.jena.riot.writer.JsonLD10Writer.serialize(JsonLD10Writer.java:177) ~[ldio-archive-file-in-jar-with-dependencies.jar:4.10.0]
2024-03-13 11:08:14 ... 41 common frames omitted

```


expected: [json-ld playground](https://json-ld.org/playground/#startTab=tab-framed&json-ld=%7B%22%40context%22%3A%5B%22https%3A%2F%2Fvocabulary.uncefact.org%2Funece-context.jsonld%22%2C%22https%3A%2F%2Fgeojson.org%2Fgeojson-ld%2Fgeojson-context.jsonld%22%2C%7B%22isVersionOf%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FisVersionOf%22%2C%22generatedAtTime%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2Fns%2Fprov%23generatedAtTime%22%2C%22%40type%22%3A%22http%3A%2F%2Fwww.w3.org%2F2001%2FXMLSchema%23dateTime%22%7D%7D%5D%2C%22type%22%3A%22LogisticsLocation%22%2C%22isVersionOf%22%3A%22unece%3Aidentifier%2Fnode%2F1%22%2C%22generatedAtTime%22%3A%222020-05-20T10%3A00%3A00Z%22%2C%22id%22%3A%22unece%3Aidentifier%2Fnode%2F1%2F1%22%2C%22name%22%3A%22k1700%22%2C%22statusCode%22%3A%220%22%2C%22geometry%22%3A%7B%22type%22%3A%22Point%22%2C%22coordinates%22%3A%5B4.9041%2C52.3676%5D%7D%2C%22servicingSpecifiedParty%22%3A%7B%22type%22%3A%22LocationParty%22%2C%22id%22%3A%22unece%3Aidentifier%2FservicingSpecifiedParty%2F1%22%2C%22name%22%3A%22DP-World%22%7D%7D&frame=%7B%22%40context%22%3A%22https%3A%2F%2Fgeojson.org%2Fgeojson-ld%2Fgeojson-context.jsonld%22%2C%22%40type%22%3A%22https%3A%2F%2Fvocabulary.uncefact.org%2FLogisticsLocation%22%7D) (might have to add a newline to the frame to trigger refresh of the framed output)

**Comment by Yalz** [comment]
Fix will be available in the upcoming 2.3.0 release. For early access, [this development build](https://github.com/Informatievlaanderen/VSDS-Linked-Data-Interactions/pkgs/container/ldi-orchestrator/190555126?tag=20240313161834) is available
